JavaScriptSPA/index.html

```diff
@@ -65,71 +90,79 @@ <h4 id="WelcomeMessage"></h4>
         xmlHttp.send();
     }
 
+
     function graphAPICallback(data) {
-        //Display user data on DOM
-        var divWelcome = document.getElementById('WelcomeMessage');
-        divWelcome.innerHTML += " to Microsoft Graph API!!";
-        document.getElementById("json").innerHTML = JSON.stringify(data, null, 2); 
+        document.getElementById("json").innerHTML = JSON.stringify(data, null, 2);
     }
 
     function showWelcomeMessage() {
         var divWelcome = document.getElementById('WelcomeMessage');
-        divWelcome.innerHTML += 'Welcome ' + myMSALObj.getUser().name;
+        divWelcome.innerHTML = 'Welcome ' + myMSALObj.getAccount().userName + "to Microsoft Graph API";
         var loginbutton = document.getElementById('SignIn');
         loginbutton.innerHTML = 'Sign Out';
         loginbutton.setAttribute('onclick', 'signOut();');
     }
 
-    // This function can be removed if you do not need to support IE
-    function acquireTokenRedirectAndCallMSGraph() {
+
+   // This function can be removed if you do not need to support IE
```

Comment:
Now that we are using a loginType flag, can we remove this comment or update it to indicate this is used for redirect flow.